src/README.md

```diff
@@ -1,15 +1,120 @@
-The docs are built on GitHub Actions and run the ``.ci_scripts/update_docs`` script.
+The docs are built on GitHub Actions and run the `.ci_scripts/update_docs` script.
 
 To build the docs locally, follow the steps mentioned below:
- 1.  Clone the repository into your local machine.
- 2.  Go into the main folder and run the following commands.  
- 3. ``conda env create -f ./.ci_scripts/environment.yml``
- 4. ``conda activate conda-forge-docs``
- 5. ``cd newsfeed && pip install --no-deps .``
- 6. ``cd ../src``
- 7. ``make html``
+
+1.  Clone the repository into your local machine.
+2.  Go into the main folder and run the following commands.
+
+```
+conda env create -f ./.ci_scripts/environment.yml
+```
+
+```
+conda activate conda-forge-docs
+```
+
+```
+cd newsfeed && pip install --no-deps .
+```
+
+```
+cd ../src
+```
+
+```
+make html
+```
 
 Documentation update PRs should only change the files in ``src``. 
 
 Once the PR is merged to ``master``, you can find the auto-generated html files in ``docs`` directory.
 
+
+### Code of conduct
+
+We adheres to the [NumFOCUS Code of Conduct](https://numfocus.org/code-of-conduct).
+
+1. Be kind to others. Do not insult or put down others. Behave professionally. 
+2. Remember that harassment and sexist, racist, or exclusionary jokes are not appropriate for conda-forge.
+3. All communication should be appropriate for a professional audience including people of many different backgrounds. Sexual language and imagery is not appropriate.
+4. conda-forge is dedicated to providing a harassment-free community for everyone, regardless of gender, sexual orientation, gender identity and expression, disability, physical appearance, body size, race, or religion. We do not tolerate harassment of community members in any form.
+5. We strive to Be empathetic, welcoming, friendly, and patient, Be collaborative, Be inquisitive, Be careful in the words that we choose.
+
+Thank you for helping make this a welcoming, friendly community for all.
+
+This is the short version of Code of Conduct. If you want to read more about it check out on [NumFOCUS Code of Conduct](https://numfocus.org/code-of-conduct).
```

Comment:
Comma needed between "it" and "check".

Also replace "on" after "check out" with "the".

"If you want to read more about it, check out the NumFOCUS Code of Conduct."